src/plugins/GUITestBase/src/GUITestBasePlugin.cpp

```diff
@@ -1706,6 +1708,7 @@ void GUITestBasePlugin::registerTests(UGUITestBase *guiTestBase) {
     REGISTER_TEST(GUITest_regression_scenarios::test_6954);
     REGISTER_TEST(GUITest_regression_scenarios::test_6959);
     REGISTER_TEST(GUITest_regression_scenarios::test_6960);
+    REGISTER_TEST_NOT_FOR_LINUX(GUITest_regression_scenarios::test_6963);
```

Comment:
Can we add a check for Linux too? We can check Fusion theme there - it is a nice looking team.

Reply:
Done